content.md

```diff
@@ -56,6 +56,7 @@ This list is also available online at: https://resources.osucord.moe/
 - [osu!Collector](https://osucollector.com/) - Beatmap collections from the members of the osu! community
 - [OMDB](http://omdb.nyahh.net/descriptors/) - Beatmaps categorized by various descriptors such as gimmicks, styles, skillsets, and more
 - [osu!pps](https://osu-pps.com/#/osu/maps) - Website for finding common farm maps within your skill range
+- [osu! Beatmap Atlas](https://osu-atlas.ameo.dev/) - An embedding-based tool that groups similar beatmaps together in space
```

Comment:
```suggestion
- [osu! Beatmap Atlas](https://osu-atlas.ameo.dev/) - A map that groups similar beatmaps together in space
```
I don't think "embeddings" mean anything to the general userbase, so I suggest simplifying it to "a map".